### PR TITLE
Remove unused translator_comment from suggestion model

### LINKED ISSUE #6527: Remove translator_comment from suggestion
not clear if this is a historical artefact but suggestion has a translator_comment field which afaict isnt used

### PR DESCRIPTION
**Pull request by phlax**
this is not currently used anywhere afaict

fixes #6527

### CODE REVIEW

File: pootle/apps/pootle_store/models.py [hunk 1]
```diff
@@ -141,27 +141,13 @@ def _target(self, value):
...
-    def translator_comment(self, value):
-        self.translator_comment_f = value
-        self._set_hash()
-
     # # # # # # # # # # # # # #  Methods # # # # # # # # # # # # # # # # # # #
 
     def __unicode__(self):
         return unicode(self.target)
 
     def _set_hash(self):
```
Comment (unho) [inline pootle/apps/pootle_store/models.py]: What is left of this method now can be folded into the single place where it is used, isn't it?

Comment (phlax) [inline pootle/apps/pootle_store/models.py]: perhaps, ill check...

Comment (phlax) [inline pootle/apps/pootle_store/models.py]: updated

**Comment by unho** [comment]
lgtm

**Pull request merged**